packages/actionmenu/package.json

```diff
@@ -30,7 +30,7 @@
   "peerDependencies": {
     "@pluralsight/ps-design-system-normalize": "^3.0.24",
     "glamor": "^2.20.0",
-    "react": ">=0.15.0 < 17.0.0"
+    "react": ">=0.16.3 < 17.0.0"
```

Comment:
These are all the packages where Theme is used, correct?

It's funny that one of these HoCs that is used internally was one of the first to get the react@16 treatment.  I imagined more that react@16 would be set as a peerDep more on a per-package basis, when that package wanted to use one of those features.

I think it's no big deal to have a bunch of packages bump like this, because teams as essentially off react 15.  But for me, it does illuminate the coupling that we have between many packages in the list.

Reply:
yeah the theme change basically bumped the version on most everything. i updated the root level react version that we use for storybook/testing as well